### PR TITLE
On topic, off topic and discouraged topics.

### PR DESCRIPTION
**Pull request by n1c**

### CODE REVIEW

**Comment by schalkventer** [comment]
Happy with the @n1c. We might need to codify this eventually, but a more broad strokes approach is probably good enough for now. 👍

**Review by schalkventer — APPROVED** [review]

**Review by zoidyzoidzoid — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -23,6 +23,44 @@ Eschew flamebait. Don't introduce flamewar topics unless you have something genu
...
+
+These off-topic spaces are valuable for networking, community building, and providing a more relaxed environment for members to interact.
+
+#### Discouraged topics
+
+Some topics, while potentially interesting, have shown a tendency to lead to heated debates and conflicts within our community. To maintain a positive and productive environment, we discourage extensive discussions on:
+
+* Politics
+* Religion
+* Controversial current events (e.g. COVID-19)
```
Comment (parautenbach) [inline README.md:48]: Climate change is apparently controversial too. I can grasp that the intention here is probably not to have an exhaustive list, but I'm calling it out nonetheless.

**Comment by schalkventer** [comment]
@violinofsmall I appreciate you referencing our previous conversation. However, I do think you are (I hope unintentionally) misconstruing some of it. So I just want to take a moment to add my perspective to the above.

To include the original question here directly for easier reference:

> So for example saying “I think Trump was an ok president” and being punished for it is against that no?

To which I responded: 

> I can't imagine how you would be punished for that? I think you conflate punished === discouraged. There are shades of gray here

Surely you can understand that conflating `saying “I think Trump was an ok president”` with setting both your profile picture and username to something politically charged is a pretty big leap. Given that, as per my own admission, there are shades of grey here, the latter is pretty clearly not aiding in the following:

> It discourages extensive discussions on these topics.

I mean it literally shows up next to every single post you make (and retroactively all you have made in the past). I don't think it gets more extensive than that.

**Comment by schalkventer** [comment]
@violinofsmall Please re-read my comment.

>I appreciate you referencing our previous conversation. However, I do think you are misconstruing some of it. [...] Surely you can understand that conflating saying “I think Trump was an ok president” with setting both your profile picture and username to something politically charged is a pretty big leap [...] We are happy to discuss your above example on it's own merits, but **I don't think it is fair conflating it with the initial discussion you referenced**.

I'm not arguing the merits of your claim. I make that clear. I'm saying the original conversation you cite says something different than you are arguing for here.

**Comment by schalkventer** [comment]
Sean, you are expecting me to speak on behalf of the entire admin team - and getting frustrated that I don't. 

It might not be clear to you, but the admin team is made up of various volunteers with different views and even different levels of commitment. Even amongst the team itself there are a lot of longstanding disagreements and a lot of give and take/compromise. 

Look, there are various avenues where you can address the entire team as a whole - yet you raise none of these issues there, the only comment I see from you is `inb4 this has little effect.`. If you address me directly I can only speak to the part that pertains me.  

I hope you can appreciate how frustrating this exchange is to me.

**Comment by schalkventer** [comment]
OK. Noted.

**Comment by JohnRoux** [comment]
Just wanted to add a little bit of info as a way to step back:

To be clear here, this is a private slack group.
n1c can do what he wants, kick out everyone who doesn't agree with him or whatever he wants.

This slack group is now comprised of over 17000 accounts.

The rest of the admin team are just helpers, who help guide and shape but have no real control or ability to change things.

We're slowly losing these helpers, which puts more work on n1c's shoulders. That's not sustainable either and less good people helping steer means that the ship is more and more likely to steer in a direction that doesn't align with the crew. 

So let's try be nice to the admins🙏

**Pull request merged**